back/node_watcher/kubernetes/nomidotwatcher-last-50k-job.yaml

```diff
@@ -27,11 +27,13 @@ spec:
       restartPolicy: Never
       containers:
         - name: nomidotwatcher
-          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:d5554bdf39b08676689423b0484920045ca99583fb0eb514c089adecbf6bcd10
+          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:1553bb0e0c492b853155816fd6b25236be4b554303fa18fbb5f8aea43ee9f397
           imagePullPolicy: Always
           env:
             - name: PRISMA_ENDPOINT
               value: http://10.0.9.43:4466
+            - name: ARCHIVE_NODE_ENDPOINT
+              value: ws://polkassembly-rpc-internal-0.parity-prod.parity.io:9944
```

Comment:
this should be `wss` right?

Reply:
SSL isn't enabled for this endpoint (yet?)

Reply:
resolved: the node is not set up with wss.